### PR TITLE
CBO-826 Apply accessibility updates to all shared views

### PR DESCRIPTION
**Pull request by naseberry**
#### What
Accessibility and coding standard updates to shared views.

#### Ticket
[CBO-826](https://dsdmoj.atlassian.net/browse/CBO-826)

#### Why
To make this section of CCCD accessible for all users.

#### How
- Ensure elements have sufficient color contrast
- Ensure table headers in a data table refers to data cells
- Improve coding standard

### CODE REVIEW

File: app/views/shared/_claim_history.html.haml [hunk 1]
```diff
@@ -6,8 +6,8 @@
   - message_controls_url = show_message_controls_external_users_claim_path(claim)
 
 .messages-container.no-print
-  .messages-list
-    = render partial: 'shared/history_items_list', locals: {claim: claim }
+  .messages-list{ 'aria-label': t('shared.claim_accordion.h2_messages'), role: 'region', tabindex: 0 }
```
Comment (jsugarman) [inline app/views/shared/_claim_history.html.haml:9]: **not a blocker**. does the claim_accordion still exist. is this locales entry just an outdated name now?

File: app/views/shared/_claim_state_transition_reasons.html.haml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   .notice
     %i.icon.icon-important
       %span.visuallyhidden
-        = "Warning"
+        = t('common.warning')
```
Comment (jsugarman) [inline app/views/shared/_claim_state_transition_reasons.html.haml:5]: 🏅

File: app/views/shared/_determinations_table.html.haml [hunk 3]
```diff
@@ -5,18 +5,18 @@
   = render partial: 'shared/summary_totals', locals: { claim: claim }
 
   %h3.heading-medium
-    = "Assessment summary"
+    = t('shared.determinations_form.assessment_summary')
```
Comment (jsugarman) [inline app/views/shared/_determinations_table.html.haml:8]: 🏅

File: app/views/shared/_evidence_list.html.haml [hunk 4]
```diff
@@ -5,7 +5,7 @@
     %h3.heading-medium
       = t('.existing_evidence')
       - if current_user_persona_is?(CaseWorker) && @claim.documents.any?
-        = link_to 'Download all', download_zip_case_workers_claim_path(@claim), class: 'font-small', style: 'float:right'
+        = link_to 'Download all', download_zip_case_workers_claim_path(@claim), class: 'font-small', style: 'float: right'
```
Comment (jsugarman) [inline app/views/shared/_evidence_list.html.haml]: **not a blocker**, could addd translation for `Download all`

**Review by jsugarman — APPROVED** [review]
👍 nice tidy 🎖 of the markup and translations. I have not checked this visually so I assume yourself and PM has.

File: app/views/shared/_claim_history.html.haml (hunk 1 above)
Comment (naseberry) [reply app/views/shared/_claim_history.html.haml:9]: Yes, claim_accordion exists. Please note we have a ticket to refactor the locales file.

**Pull request merged**